cyclonedx/model/bom.py

```diff
@@ -217,6 +258,9 @@ class Bom:
 
     Once you have an instance of `cyclonedx.model.bom.Bom`, you can pass this to an instance of
     `cyclonedx.output.BaseOutput` to produce a CycloneDX document according to a specific schema version and format.
+
+    @todo: Handle different Spec Versions during (de-)serialization
```

Comment:
This may still be, but as we have a ton of tests that include (de-)serializing to/from all versions of the SPEC, this may not actually be a @todo anymore...

Will review.

Reply:
No longer a todo. Removed.